sdk/src/main/scala/io/horizen/storage/SidechainSecretStorage.scala

```diff
@@ -153,4 +153,25 @@ class SidechainSecretStorage(storage: Storage, sidechainSecretsCompanion: Sidech
     storage.update(new ByteArrayWrapper(Utils.nextVersion), updateList, removeList)
     this
   }
+
+  def storeSecretAndNonceAtomic(secret: SidechainTypes#SCS, nonce: Int, keyTypeSalt: Array[Byte]): Try[SidechainSecretStorage] = Try  {
```

Comment:
It seems that `storeNonce` method is not used anymore so maybe it is better to remove it.

Reply:
Done